public/calendar_reboot.js

```diff
@@ -911,8 +909,9 @@ var LocalCalendarStorage = function(params) {
     self.endWork = function() {
         // Will decrement the counter and maybe fire an event.
         
+        
         self.activeCalls -= 1;
-        console.log("store's endWork is running, activeCalls = " + self.activeCalls);
+        if (self.activeCalls < 0) console.error("No work has been started");
```

Comment:
It is best not to get into the habit of leaving out the curly braces. The benefit is very small and the likelihood of introducing bugs accidentally is very high. https://www.reddit.com/r/programming/comments/22ghj1/the_heartbleed_bug/